applications/vanilla/settings/class.hooks.php

```diff
@@ -367,6 +367,15 @@ public function base_render_before($sender) {
         if ($sender->Menu) {
             $sender->Menu->addLink('Discussions', t('Discussions'), '/discussions', false, ['Standard' => true]);
         }
+        
+        if (!inSection('Dashboard')) {
+            // Spoilers assets
+            $sender->addJsFile('spoilers.js', 'dashboard');
+            $sender->addCssFile('spoilers.css', 'dashboard');
+            $sender->addDefinition('Spoiler', t('Spoiler'));
+            $sender->addDefinition('show', t('show'));
+            $sender->addDefinition('hide', t('hide'));
+        }
```

Comment:
This seems like it should be in `/applications/dashboard/settings/class.hooks.php` instead.